### DISCUSSION TITLE
Add a default language setting in GitHub profile for Codespaces

### DISCUSSION THREAD

**Discussion by sinedied** [community/community#67847] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Currently, Codespaces seems to default to a language related to my location (IP?) every time I create a new one.

For me, it forces VS Code in French (as I live in France), whereas my OS, browser and even GitHub are set to US/English. This is very annoying, and I have to change the language and restart the container every time.

I would love to have an option in my GH account to set the default language to be used in Codespaces, or at least make it default to the browser language instead.

I've seen this issue (https://github.com/orgs/community/discussions/52603), but as for me even with my OS and browser set to english it's not working.

**Comment by emonget** [comment]
Any update on that? Same for me,  many git commands are unrecognizable in french native language.
English is so common in DEV that I don't see the point of defaulting to local language.

**Reply by RupertBarrow** [reply]
Pareil !

**Comment by FLAK-ZOSO** [comment]
So there is still no way to change this? Italian is pretty confusing for Visual Studio Code, not to speak of Estonian.

**Comment by saynotoid** [comment]
agree - id like to have English by default :(:(
